<launch>
	<arg
		name="device_name"
		default="filtration_system"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyACM0"/>
	<arg
		name="simulation"
		default="False"/>
	<node
		name="filtration_system"
		pkg="roslabware_drivers"
		type="filtration_system"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)"
		output="screen"/>
</launch>
